libs/core/src/components/pds-table/stories/pds-table.stories.js

```diff
@@ -0,0 +1,227 @@
+import { html } from 'lit-html';
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  argTypes: extractArgTypes('pds-table'),
+  component: 'pds-table',
+  decorators: [withActions],
+  parameters: {
+    actions: {
+      handles: ['onclick', 'pdsTableRowSelected'],
+    },
+  },
+  title: 'components/Table',
+};
+
+const BaseTemplate = (args) => html`
+<pds-table
+  compact="${args.compact}"
+  component-id="${args.componentId}"
+  fixed-column="${args.fixedColumn}"
+  responsive="${args.responsive}"
+  selectable="${args.selectable}"
+>
+  <pds-table-head>
+    <pds-table-head-cell class="${args.compact ? 'hydrated is-compact' : 'hydrated'}">Column Title</pds-table-head-cell>
```

Comment:
Does `is-compact` need to be passed to each table cell manually like this? If so, why? 

1. Could we find a way to improve this for the developer experience? 
2. Are there instances where a single cell may not use `is-compact`?

Reply:
It does not, this was an attempt to get storybook controls to work. Will mark this one as resolved based on the convo below. The compact only has to be added to the table and will handle all the cells.